docs/modules/administration/pages/image-lifecycle.adoc

```diff
@@ -0,0 +1,51 @@
+= Image Lifecycles
+
+ODS makes use of and provides a few container images. This document will detail which container images are used, how contaienr images are built during an ODS installation, and how images are consumed by users of ODS.
+
+== Categories of images
+
+There are three broad categories of images:
+
+1. Images used in the central ODS namespace
+2. Images used in the `*-cd` namespace of each ODS project
+3. Application images used in the `*-dev` and `*-test` namespace
+
+Every category has different lifecycles.
+
+=== Images used in the central ODS namespace
+
+Those images are owned by ODS admins. Examples of such images include the SonarQube image and the Nexus image. All of the images used by containers in the central ODS namespace are built by ODS during its installation. No container uses a vanilla image as provided by e.g.  DockerHub or quay.io.
```

Comment:
I'm not totally certain about what you want to convey with this message: ` No container uses a vanilla image as provided by e.g.  DockerHub or quay.io.` Do you mean that the images present in the `ods` namespace do not use any base image coming from a public registry like Dockerhub?

Reply:
Hmm yea - I meant that all deployments use images from local image streams. No deployment pulls directly from DockerHub.